packages/apps/marmalade-marketplace/src/pages/mytokens/index.tsx

```diff
@@ -7,24 +7,34 @@ import { Grid, GridItem, Stack, Heading } from "@kadena/kode-ui";
 
 export default function MyTokens() {
   const [tokens, setTokens] = useState<Array<NonFungibleTokenBalance>>([]);
-  const { webauthnAccount } = useAccount();
+  const { account } = useAccount();
 
   const fetchTokens = async (accountName?:string) => {
     if (!accountName) return;
 
     const tokens = await getTokens(accountName);
-    console.log(tokens)
+
     setTokens(tokens.filter(token => token.balance > 0));
   };
 
   useEffect(() => {
-    fetchTokens(webauthnAccount?.account);
-  }, [webauthnAccount?.account]);
+    fetchTokens(account?.accountName);    
+  }, [account?.accountName]);
 
   return (
     <Stack flex={1} flexDirection="column">
-      <Heading as="h1">My Tokens</Heading>
+      <div style={{ marginTop: '-60px', marginLeft: '-6%', marginRight: '-6%'}}>
```

Comment:
At somepoint you'd probably want to refactor this to use `Stack` and `Image`. Specially Image is very useful as it provides optimizations for faster initial loads.